filter.go

```diff
@@ -0,0 +1,112 @@
+package reviewdog
+
+import (
+	"path/filepath"
+	"strings"
+
+	"github.com/haya14busa/reviewdog/diff"
+)
+
+type FilteredCheck struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type FilteredCheck should have comment or be unexported